src/classes/STG_SettingsManager_TEST.cls

```diff
@@ -575,12 +596,14 @@ public with sharing class STG_SettingsManager_TEST {
     * @description Tests the Recurring Donations panel
     * @return void
     */
-    public static testMethod void testPanelRD() {
+    @isTest
+    public static void testPanelRD() {
```

Comment:
Thank you for updating old test methods. Can you please update them to `private` as well if they are not used outside of this test class?